.github/workflows/autoformat.yml

```diff
@@ -0,0 +1,22 @@
+name: Autoformat code
+on: pull_request
+
+# This action only need a single permission in order to autoformat the code.
+permissions:
+  contents: read
+
+jobs:
+  autoformat-code:
+    name: Autoformat code
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: 'Autoformat'
+      uses: rolfbjarne/autoformat@v0.1
+      with:
+        projects:
+            - "Maui/MLTrainer/MLTrainer.csproj"
```

Comment:
@rolfbjarne I'm not sure I've don't this right. In your test example you did this

```
 projects: "testproject/testproject.csproj testproject/testproject.csproj"
```
which seemed a bit odd to have the same project twice. So I figured I'd try this format instead.

Reply:
> @rolfbjarne I'm not sure I've don't this right. In your test example you did this
> 
> ```
>  projects: "testproject/testproject.csproj testproject/testproject.csproj"
> ```
> 
> which seemed a bit odd to have the same project twice. So I figured I'd try this format instead.

Yeah, there are two projects because I wanted to verify that using more than one project worked (and I only had one project to test with at the time, so I added the same project twice). Everything worked as expected, but I'll remove that from the test example, since it's confusing.

Reply:
> Can these target `.sln` files?
> 
> Like:
> 
> * [`main`/Maui/MLTrainer/MLTrainer.sln](https://github.com/jonathanpeppers/inclusive-code-reviews-ml/blob/main/Maui/MLTrainer/MLTrainer.sln?rgh-link-date=2022-11-01T13%3A36%3A18Z)
> * [`main`/ml.net/InclusiveCodeReviews.sln](https://github.com/jonathanpeppers/inclusive-code-reviews-ml/blob/main/ml.net/InclusiveCodeReviews.sln?rgh-link-date=2022-11-01T13%3A36%3A18Z)
> 
> This would help if we add new projects, won't have to add them to this list.

Yes, `.sln` files can be the target too.

Reply:
ok done . should be good now